docker-compose-dev.yml

```diff
@@ -16,26 +16,6 @@ services:
     volumes:
       - contracts:/contracts
 
-  forum:
-    image: appliedblockchain/node-bb-atrium
-    build:
-      context: packages/node-bb
-      dockerfile: Dockerfile
-    depends_on:
-      - mongo
-    ports:
-      - 4567:4567
-
-  remix:
-    image: appliedblockchain/remix-atrium
-    build:
-      context: packages/remix
-      dockerfile: Dockerfile
-    depends_on:
-      - parity1
-    ports:
-      - 8080:80
-
```

Comment:
These should be kept if you don't want to spin them up you can spin up a subset of the components:
```docker-compose up init parity1 ui mongo```